src/main/java/com/commercetools/sync/categories/helpers/CategoryReferenceResolver.java

```diff
@@ -60,8 +60,8 @@ public CategoryReferenceResolver(@Nonnull final CategorySyncOptions options,
      * taken from the id field of the references.
      *
      * @param categoryDraft the categoryDraft to resolve it's references.
-     * @return a {@link CompletionStage} that contains as a result a new categoryDraft instance with resolved category
-     *          references or, in case an error occurs during reference resolution,
+     * @return a {@link CompletionStage} that contains as a result of a new categoryDraft instance with resolved
```

Comment:
`that contains as a result of a new categoryDraft..` -> `that contains, as a result, a new categoryDraft..`

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/319/commits/219b247416983efe24dada2efdaa3984a5abdbfa